src/utils/images.ts

```diff
@@ -1,4 +1,10 @@
 const ALLOWED_HOSTS = ['https://media.guim.co.uk/', 'https://i.guim.co.uk/'];
+const ALLOWED_DEV_HOSTS = [
+    ...ALLOWED_HOSTS,
+    'https://s3-eu-west-1.amazonaws.com/media-origin.test.dev-guim.co.uk/',
```

Comment:
We should raise a ticket with Ed Tools to get object listing removed from this bucket, especially as we have now made the bucket name public...

Reply:
Great shout. We should also check that it's ok to make this bucket name public at all. We might wanna nuke the history here if not. My suspicion is that this bucket name is "private", in the three-tier "public", "private", "secret" categorisation, which means it shouldn't be in a public repo.

We're racking up use cases that need secrets, I wanna chat to you all about adding a backend to this where we can put them.

Reply:
Ahh thanks for pointing this out, that concern didn't occur to me here. Let's chat about what needs to happen with this.